packages/base-zone/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@agoric/base-zone",
+  "version": "1.0.0",
+  "description": "Allocation zone abstraction library and heap implementation",
+  "type": "module",
+  "repository": "https://github.com/Agoric/agoric-sdk",
+  "main": "./src/index.js",
+  "scripts": {
+    "build": "exit 0",
+    "test": "ava",
+    "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
+    "test:xs": "exit 0",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc -p jsconfig.json",
+    "lint:eslint": "eslint ."
+  },
+  "exports": {
+    ".": "./src/index.js",
+    "./heap.js": "./heap.js",
```

Comment:
Why have root heap.js? Exporting src/heap.js has the same API.

If there's a good reason to have a root file for each export, why not do the same for index?

Reply:
There is a reason (engines or bundlers that don't support `"exports"`), and "src/index.js" is not necessary because it already appears in `"main"` (which has wide support).